docs/contracts/v2/reference/smart-contracts/08-deployment-addresses.md

```diff
@@ -5,14 +5,15 @@ title: V2 Deployment Addresses
 
 # Uniswap v2 Contract Deployments
 
-Contract addresses for the [Uniswap V2 Factory](https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Factory.sol) 
+Contract addresses for the [`Uniswap V2 Factory`](https://github.com/Uniswap/v2-core/blob/master/contracts/UniswapV2Factory.sol) and the [`Uniswap V2Router02`](https://github.com/Uniswap/v2-periphery/blob/master/contracts/UniswapV2Router02.sol)
 
-| Network                                              | Contract Address                                |
-| ---------------------------------------------------- | ----------------------------------------------- |
-| Mainnet, Ropsten, Rinkeby, Goerli, Kovan             | `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f`    |
-| Arbitrum                                             | `0xf1D7CC64Fb4452F05c498126312eBE29f30Fbcf9`    |
-| Avalanche                                            | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C`    |
-| BNB Chain                                            | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6`    |
-| Base                                                 | `0x8909Dc15e40173Ff4699343b6eB8132c65e18eC6`    |
-| Optimism                                             | `0x0c3c1c532F1e39EdF36BE9Fe0bE1410313E074Bf`    |
-| Polygon                                              | `0x9e5A52f57b3038F1B8EeE45F28b3C1967e22799C`    |
+| Network                                              | Factroy Contract Address                     | V2Router02 Contract Address                  |
+| ---------------------------------------------------- | -------------------------------------------- | -------------------------------------------- |
+| Mainnet, Ropsten, Rinkeby, Goerli, Kovan             | `0x5C69bEe701ef814a2B6a3EDD4B1652CB9cc5aA6f` | `0x7a250d5630B4cF539739dF2C5dAcb4c659F2488D` |
```

Comment:
Do we still need to list the deprecated testnets?

Reply:
Good point. I'll remove all but Goerli, which still existing for now and only available v2

Reply:
Hmmm.. Goerli validators stopped producing blocks for a while https://goerli.etherscan.io/, although that precedes EF's planned deprecation date

Reply:
And I actually had to disable Goerli from routing all together because of that. https://github.com/Uniswap/routing-api/pull/500

Reply:
Doh, yeah, well we knew this was coming. Will have to push FE to update the supported list.